pages/task/read/routers/read.js

```diff
@@ -208,6 +208,16 @@ module.exports = () => {
     next();
   });
 
+  app.use((req, res, next) => {
+    if (req.task.data.model === 'project') {
+      const deadline = get(req.task, 'data.deadline');
+      const isExtended = get(deadline, 'isExtended', false);
+      const deadlineDate = get(deadline, isExtended ? 'extended' : 'standard');
+      res.locals.static.daysSinceDeadline = daysSinceDate(deadlineDate);
```

Comment:
We already have `task.data.deadline` as an object for storing deadline-related metadata. Does it make sense for this to be a property on that object?

Reply:
Are you talking about adding it to the deadline object in the workflow decorator, or just patching it in the UI to avoid the extra locals var? All the current properties on the deadline object (standard, extended, isExtended, isExtendable) are static unless someone performs an action (e.g. resubmits the task / extends the deadline) - do we have any other `task.data.x` props that are dynamic?